### PR TITLE
fix lint_error_index.js

### PR DESCRIPTION
**Pull request by Vladyslav-Shvorobuk**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Look pretty. Nice job!

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,148 @@
...
+
+    return result;
+  }
+
+  sort(callback) {
+    if (arguments.length === 1 && typeof callback === 'function') {
+      for (let i = 0; i < this.length - 1; i++) {
+        for (let j = 0; j < this.length - 1; j++) {
+          if (callback(this[j], this[j + 1]) > 0) {
+            const max = this[j];
```
Comment (ZelikSV) [inline src/index.js:128]: Will be Error. Change const max to let

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,148 @@
...
+            this[j] = this[j + 1];
+            this[j + 1] = max;
+          }
+        }
+      }
+    } else if (arguments.length === 0) {
+      for (let i = 0; i < this.length - 1; i++) {
+        for (let j = 0; j < this.length - i; j++) {
+          if (`${this[j]}` > `${this[j + 1]}`) {
+            const max = this[j];
```
Comment (ZelikSV) [inline src/index.js:138]: Will be error. Change const max to let

File: src/index.js (hunk 1 above)
Comment (Vladyslav-Shvorobuk) [reply src/index.js:128]: The value of the variable "const" is given once, there will be no error

**Review by ZelikSV — APPROVED** [review]
All changes rewriting. Good job

**Pull request merged**